_news/2018-06-07-forseti-hackday.md

```diff
@@ -2,14 +2,19 @@
 title:  Forseti Hack Day
 author: Justin Jawad
 ---
-Google is hosting the first Forseti Security Hack Day on July 23rd. Thank you to all who registered! We have had a very positive international response from participants representing leading organizations.
+Google is hosting the first Forseti Security Hack Day on July 23rd. Thank you to
+all who registered! We have had a very positive international response from
+participants representing leading organizations.
+
+Come hungry! We have a full agenda planned plus breakfast and lunch a la Google
+with speakers presenting on a variety of cloud security-related topics. Dive
+into Forseti with us and learn more about how to:
 
-Come hungry! We have a full agenda planned plus breakfast / lunch a la Google with speakers presenting on a variety of cloud security-related topics. Dive into Forseti with us and learn more about how to :
 * Keep track of your environment.
 * Monitor and understand your current Google Cloud Platform security policies.
 * Provide corrective action when problems are detected.
 
-Spend the day with the Forseti team at Google, gain insight into what’s on the horizon, and help make our code base stronger together!
-
-We still have openings but they won’t last long! Register now to attend.
+Spend the day with the Forseti team at Google, gain insight into what’s on the
+horizon, and help make our code base stronger together!
 
+We still have openings but they won’t last long! [Register for Forseti Hack Day](https://docs.google.com/forms/d/1QzJHCTfG0_E4ag7NMk5e6LwVNwn81I7P-HDqJygeijg/edit) now to attend.
```

Comment:
This link is probably not the want you want to use? If it's a form there's a menu option in the form to get a link to the form, use that one instead.